index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script src="index.js"></script>
+    <link rel="stylesheet" href="index.css">
+    <title>Unit 2 Assessment</title>
+</head>
+<body>
+    <div id="banner">
+        <img id="logo" src="https://upload.wikimedia.org/wikipedia/en/thumb/c/ca/Studio_Ghibli_logo.svg/1200px-Studio_Ghibli_logo.svg.png">;
+        <h1 id="reviewApp">Ghilbi Review App</h1>
+    </div>
+    <div id="middle">
+    <select id="dropMenu" onchange="switchMovies()">
```

Comment:
Adding and `onchange` attribute in the HTML has the same effect as adding a listener with `addEventListener()`. The later is preferred in our case. It is not good practice to mix the two approaches. Change this to be done with  `addEventListener()` instead.